src/api/api.js

```diff
@@ -0,0 +1,6 @@
+const BASE_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=91e17a04&t=';
```

Comment:
```suggestion
const BASE_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=91e17a04&t=';
```
You just need to correct this :)

Reply:
It worked out!) thanks a lot)